pkg/config/database.go

```diff
@@ -189,28 +189,16 @@ func unknownDbType(t string) error {
 //
 // https://mariadb.com/kb/en/galera-cluster-system-variables/#wsrep_sync_wait
 func setGaleraOpts(ctx context.Context, conn driver.Conn, wsrepSyncWait int64) error {
-	const galeraOpts = "SET SESSION wsrep_sync_wait=?"
+	galeraOpts := "SET SESSION wsrep_sync_wait=" + strconv.FormatInt(wsrepSyncWait, 10)
 
-	stmt, err := conn.(driver.ConnPrepareContext).PrepareContext(ctx, galeraOpts)
+	_, err := conn.(driver.ExecerContext).ExecContext(ctx, galeraOpts, nil)
 	if err != nil {
 		if errors.Is(err, &mysql.MySQLError{Number: 1193}) { // Unknown system variable
 			return nil
 		}
```

Comment:
@glbyers' [patch](https://github.com/Icinga/icingadb/issues/750#issuecomment-2073765560) also swallowed [`driver.ErrSkip`](https://pkg.go.dev/database/sql/driver#ErrSkip). Even as the [`driver.ExecerContext` documentation](https://pkg.go.dev/database/sql/driver#ExecerContext) states that it might return this error, I am not sure why it was included in the patch. Maybe @glbyers can enlighten us?

Reply:
Precisely speaking, it's this error that patch had swallowed: https://github.com/go-sql-driver/mysql/blob/d7ddb8b9e324830b1ede89c5fea090c824497c51/connection.go#L318

I also have no idea why to swallow this error because this makes the query a complete no-op.

Reply:
> @glbyers' [patch](https://github.com/Icinga/icingadb/issues/750#issuecomment-2073765560) also swallowed [`driver.ErrSkip`](https://pkg.go.dev/database/sql/driver#ErrSkip). Even as the [`driver.ExecerContext` documentation](https://pkg.go.dev/database/sql/driver#ExecerContext) states that it might return this error, I am not sure why it was included in the patch. Maybe @glbyers can enlighten us?

Right. I'd read that ErrSkip may be raised if interpolation wasn't available in the driver, but wasn't aware that it would result in a complete noop. In our situation, we're not concerned that `wsrep_sync_wait` isn't set at all as we control writes to a single node via proxysql (so wsrep_sync_wait=0 or not at all set suits our use case). Apologies for overlooking that, but as stated, that works just fine for us. I'll test your patch with interpolation on Monday when i'm back at work.

Reply:
OK. I have applied your patch and tested. I can confirm this works as expected. ProxySQL events log captures the query;

```json
{"client":"127.0.0.1:37024","digest":"0x4935002AA513D03A","duration_us":0,"endtime":"2024-04-28 21:45:44.090140","endtime_timestamp_us":1714340744090140,"event":"COM_QUERY","hostgroup_id":-1,"query":"SET SESSION wsrep_sync_wait=0","rows_affected":0,"rows_sent":0,"schemaname":"icingadb","starttime":"2024-04-28 21:45:44.090140","starttime_timestamp_us":1714340744090140,"thread_id":7,"username":"icingadb"}
```